{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--d979322f-0792-4412-b5d4-8e3365b80f61",
    "created": "2020-05-26T20:29:46.051173Z",
    "modified": "2020-05-26T20:29:46.051173Z",
    "name": "CVE-2018-13097",
    "description": "An issue was discovered in fs/f2fs/super.c in the Linux kernel through 4.17.3. There is an out-of-bounds read or a divide-by-zero error for an incorrect user_block_count in a corrupted f2fs image, leading to a denial of service (BUG)."
}